Directory.Build.props

```diff
@@ -39,7 +47,8 @@
 		<!-- The MSAL.snk has both private and public keys -->
 		<DelaySign>false</DelaySign>
 		<EnableNETAnalyzers>true</EnableNETAnalyzers>
-		<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

Comment:
We can simplify a bit if we don't want to have opt outs. These can go, and the conditions in the item groups can go.